src/clj/game/cards/events.clj

```diff
@@ -1199,6 +1214,12 @@
     :events {:pre-rez-cost nil}
     :end-turn {:effect (effect (unregister-events card))}}
 
+   "Spear Phishing"
+   {:implementation "Bypass is manual"
+    :prompt "Choose a server"
+    :choices (req runnable-servers)
+    :effect (effect (run target nil card))}
+   
```

Comment:
Fine as a starting point--for real flair, since this card references ICE at a defined position, you could have it register an `:encounter-ice` event where if the `ice-index` of the event target is 0, you print a message to the game log.